### ISSUE TITLE
Add metaphyseal chevron deformity

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10528]
**Preferred term label:**
Metaphyseal chevron deformity

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
The chevron sign refers to an inverted V-shape metaphyses. It is most commonly seen in the distal femur and tends to disappear with age. PMID: 25349664
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/73a0933a-9585-4495-833c-6fcd7e7ee91e)
Source: PMID25808063

**Parent term (use hpo.jax.org/app)**
Abnormal metaphysis morphology HP:0000944

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
CODAS syndrome (omim 600373), PMC6611735, PMID25808063

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000870
- label: Metaphyseal chevron deformity
- parents: [Abnormal metaphysis morphology (HP:0000944)](https://hpo.jax.org/app/browse/term/HP:0000944)
- pmid: PMID:25349664|PMID:25808063
- def: The chevron sign refers to an inverted V-shape metaphyses. It is most commonly seen in the distal femur and tends to disappear with age.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10528
- synonyms:
